yahoofantasy/resources/league.py

```diff
@@ -7,13 +7,13 @@
 from .week import Week
 from .draft_result import DraftResult
 from .transaction import Transaction
+from .player import Player
 
 
 class League:
     def __init__(self, ctx, league_id):
         self.ctx = ctx
         self.id = league_id
-        self.players = list()
```

Comment:
I couldn't find a usage for these players list. To make function naming standard (teams(), standings()), I removed these class attribute and added a method instead.